source/jormungandr/tests/check_utils.py

```diff
@@ -842,6 +842,10 @@ def is_valid_vehicle_journey(vj, depth_check=1):
         assert 'validity_pattern' not in vj
 
 
+def is_valid_trip(trip, depth_check=1):
```

Comment:
for the moment, but it will change :wink: And those methods needs to have the same signature